### ISSUE TITLE
Fix accessibility

### ISSUE THREAD

**Issue by theanxy** [geelen/x-gif#10]
The site is spreading front-end development anti-patterns such as IMG tags with no ALT attributes, poor HTML semantics, skipping Hx headings and much more. Please use WAI-ARIA to hide presentational elements from screen readers.

Attached is the output from Fangs:
![chrome___fangs_-_fangs___x-gif___http___geelen_github_io_x-gif___http___i_imgur_com_ikxh4e2_gif__and__x-gif__and_2014-03-20_1753_-_amandaguerriero_s_library](https://f.cloud.github.com/assets/92843/2486424/6393842c-b126-11e3-94a2-9e681018340d.png)

**Comment by geelen** [comment]
Yeah this was hacked up in a super rush before I got on a flight, and got a
fair bit more attention than I expected.

I think the x-gif tag should be as similar as possible to the img tag in
terms of semantics, and all the tags within it should be completely hidden.
Sound right?

This is my first polymer element too, so I'm not sure if there's a pattern
I can follow. Would be happy to receive any more feedback/PRs.

**Comment by geelen** [comment]
Started PR #11 to work on this - would love if you could work with me on fixing this @theanxy :)
